yarn-project/aztec/src/cli/cli.ts

```diff
@@ -32,7 +32,8 @@ export function injectAztecCommands(program: Command, userLog: LogFn, debugLogge
     .option('-px, --pxe [options]', cliTexts.pxe)
     .option('-a, --archiver [options]', cliTexts.archiver)
     .option('-s, --sequencer [options]', cliTexts.sequencer)
-    .option('-r, --prover [options]', cliTexts.prover)
+    .option('-r, --prover [options]', cliTexts.proverAgent)
```

Comment:
I'm not entirely convinced of the way in which we are deviating from the current cli command system. If I want to run a prover node why wouldn't I just run:

`aztec start --node --prover`

Is it solely about being able to run multiple seperate prover nodes and only have a single transaction pool that they pull from? Is this likely? Is it even necessary to run multiple prover nodes?

Reply:
This is mostly for backwards compatibility: we still support having a sequencer generate and publish proofs as it generates a block (this will be removed soon-ish), so we still need to handle the configuration of "sequencer node with local prover-agent", which is `--node --sequencer --prover`. Once we deprecate that, I agree on the need to revisit the CLI options here.